plot_data/core.py

```diff
@@ -45,6 +46,25 @@ def delete_none_from_dict(dict1):
                 dict2[key] = value
     return dict2
 
+def serialize_dates_in_list(list_: List[Any]) -> List[Any]:
+    for i, element in enumerate(list_):
+        if isinstance(element, datetime.datetime):
+            list_[i] = f"{element.timestamp() * 1000}gmt+"
+        if isinstance(element, list):
+            list_[i] = serialize_dates_in_list(element)
+        if isinstance(element, dict):
+            list_[i] = serialize_dates_in_dict(element)
+    return list_
+
+def serialize_dates_in_dict(dict_: Dict[str, Any]) -> Dict[str, Any]:
+    for (key, value) in dict_.items():
+        if isinstance(value, datetime.datetime):
+            dict_[key] =  f"{value.timestamp() * 1000}gmt+"
```

Comment:
Consider setting this as a `serialize_date` function as it duplicated for lists and dicts

Reply:
Changed